runners/flink/flink_runner.gradle

```diff
@@ -251,7 +257,7 @@ def createValidatesRunnerTask(Map m) {
     def pipelineOptionsArray = ["--runner=TestFlinkRunner",
                                 "--streaming=${config.streaming}",
                                 "--useDataStreamForBatch=${config.useDataStreamForBatch}",
-                                "--parallelism=2",
+                                "--parallelism=1",
```

Comment:
This seems like it could be a problem, actually. Why did you have to do this? Memory pressure on test workers?

Reply:
Yes I think validate runner was getting OOMKilled